#! /bin/sh /usr/share/dpatch/dpatch-run
# Description: Add translation domain to inkscape.desktop.in
#  This is an Ubuntu-specific fix to allow translations to be stripped
#  from the Desktop entry file and read from language packs instead.
#  This is only needed for backports to Ubuntu Maverick and earlier.
#  Later versions of Ubuntu handle this using dh_translations.
# Author: Alex Valavanis <valavanisalex@gmail.com>
# Forwarded: not-needed
# Last-Update: 2011-09-23


@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' ubuntu-inkscape-maverick-backport~/inkscape.desktop.in ubuntu-inkscape-maverick-backport/inkscape.desktop.in
--- ubuntu-inkscape-maverick-backport~/inkscape.desktop.in	2011-09-23 23:11:39.092461091 +0100
+++ ubuntu-inkscape-maverick-backport/inkscape.desktop.in	2011-09-23 23:12:21.834802992 +0100
@@ -13,6 +13,7 @@
 StartupNotify=true
 Icon=inkscape
 X-Ayatana-Desktop-Shortcuts=Drawing
+X-Ubuntu-Gettext-Domain=inkscape
 
 [Drawing Shortcut Group]
 _Name=New Drawing
